meetings/2021-11-24.md

```diff
@@ -0,0 +1,82 @@
+* Justin: I like the simplification and I agree that it is source of confusion. Any time in an authz system you can remove things, you reduce the opportunity for bad authz that doesn't do what the user expects. So generally I'm very much in favour of the simplification, but I'd like to take more time going through the feedback. Make sure I keep all that in mind. But TLDR; I think the simplification is nice. The bit I mentionned at the front is not necessarily something that affects me but may affect others, so I'd like to hear more context about how others might be affected. It feels like not such a hard migration path. But migrations are migrations. But hearing from a purely draft standpoint is good.
+* Henry: It seems to me that the simplification is useful. I personally got stuck on that aspect and having the simplification makes it clearer. It also seems compatible with ACR on ACR so it sounds good to me.
+* Matthieu: One more thing that might be worth mentioning is that I think we can and should align on the access modes and if everyone agrees that we need ControlRead, we can push it on the Solid Protocol level as entailing specific operations.
+* Pavlik: I have a question on auxiliary resources. I understand that only aux resources have a control on them. Is there a case on controlling an 
+* If we say there is a special access mode that refers to an auxiliary resource. It sounds to me like this approach is not using ACR on auxiliary resource. You would still need that path.
+* Matthieu: Are you asking if we have a way currently in Solid to define ACRs on ACRs?
+* Pavlik: Say you have controlRead on resource A, then you have a custom access mode to target the ACR. It doesn't apply to resource A it applies to the ACR of resource A.
+* Matthieu: My short answer is that it is not up to ACP to define the mechanism you are proposing. ACP has a simple extension point, which is access modes. Those access modes are not defined by ACP but by the protocol.  The Solid Protocol says that in order to be able to edit an auxiliary resource you need an access mode acl:Control. So solid says you need ACL control to edit the auxiliary resource.
+* Barath: Can you be in a situation where your permissions would allow you to delete a resource but not its auxiliary control resource?
+* Matthieu: First, That is in my opinion something that the SOlid Protocol should define precisely. Moreover, I think in order to delete a resource as defined in the Solid Protocol currently you need acl:Control on both the resource and its container. So short answer, it should not be possible.
+* Henry: There is actually quite a few other good things that can happen from this change. Let me think... I think what Matthieu is saying is that one could open another use case for defining ACLs on ACLs rather than control. I think that someone was arguing giving access control to sub-parts of a graph. Even though it wasn't mentionned.
```

Comment:
```suggestion
* Barath: Can you be in a situation where your permissions would allow you to delete a resource but not its auxiliary control resource?
* Matthieu: First, that is in my opinion something that the Solid Protocol should define precisely. Moreover, I think in order to delete a resource as currently defined in the Solid Protocol, you need `acl:Control` on both the resource and its container. So short answer, it should not be possible.
* Henry: There are actually quite a few other good things that can happen from this change. Let me think... I think what Matthieu is saying is that one could open another use case for defining ACLs on ACLs, rather than control. I think that someone was arguing giving access control to sub-parts of a graph. Even though it wasn't mentioned.
```